README.md

```diff
@@ -360,3 +360,57 @@ will be rendered in the left position of the `navigationBar`.
 will be rendered in the title position of the `navigationBar`.
 - `renderRight` - a function that should return a React component that
 will be rendered in the right position of the `navigationBar`.
+
+## Tabs
+
+A minimal example using tabs:
+
+```
+<TabNavigation
+  id="main"
+  initialTab="more">
+  <TabItem
+    id={'home'}
+    title={'Home'}
+    selectedStyle={styles.selectedTab}
+    renderIcon={(isSelected) => <Image source={'home-icon'} /> }>
+    <StackNavigation
+      id={id}
+      initialRoute={Router.getRoute('home')}
+    />
+  </TabItem>
+  <TabItem
+    id={'posts'}
+    title={'Posts'}
+    selectedStyle={styles.selectedTab}
+    renderIcon={(isSelected) => <Image source={'posts-icon'} /> }>
+    <StackNavigation
+      id={id}
+      initialRoute={Router.getRoute('posts')}
+    />
+  </TabItem>
+  <TabItem
+    id={'profile'}
+    title={'Profile'}
+    selectedStyle={styles.selectedTab}
+    renderIcon={(isSelected) => <Image source={'profile-icon'} /> }>
+    <StackNavigation
+      id={id}
+      initialRoute={Router.getRoute('profile')}
+    />
+  </TabItem>
+</TabNavigation>
+```
+
+If you'd like to switch tabs you can use the `jumpToTab` as below. Be sure to pass in `navigation`.
+
+```
+<TouchableOpacity
+  onPress={() => {
+    navigation.performAction(({ tabs, stacks }) => {
+      tabs('main').jumpToTab('profile');
```

Comment:
Sorry... Yes I think I did. I'll add that back..

On 12 Sep 2016 19:57, "Brent Vatne" <notifications@github.com> wrote:

> In README.md
> <https://github.com/exponentjs/ex-navigation/pull/76#discussion_r78410902>
> :
>
> > +    renderIcon={(isSelected) => <Image source={'profile-icon'} /> }>
> > +    <StackNavigation
> > +      id={id}
> > +      initialRoute={Router.getRoute('profile')}
> > +    />
> > +  </TabItem>
> > +</TabNavigation>
> > +```
> > +
> > +If you'd like to switch tabs you can use the `jumpToTab` as below. Be sure to pass in `navigation`.
> > +
> > +```
> > +<TouchableOpacity
> > +  onPress={() => {
> > +    navigation.performAction(({ tabs, stacks }) => {
> > +      tabs('main').jumpToTab('profile');
>
> did you have to set the navigatorUID prop on TabNavigation for this to
> work @elie222 <https://github.com/elie222>?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/exponentjs/ex-navigation/pull/76/files/470a44f22b53d68dcc76a89b20ce64608cfa6c51#r78410902>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AC8oXx6V5WHb3jD_rPBy0Sjr2ZsXTSTcks5qpYR-gaJpZM4J6yv4>
> .
>